data/yam/agama/auto/lib/scripts.libsonnet

```diff
@@ -0,0 +1,43 @@
+local enable_root_login = {
+  name: 'enable root login',
+  chroot: true,
+  body: |||
+    #!/usr/bin/env bash
+    echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
+  |||
+};
+
+{
+  default: {
+    post: [ enable_root_login ]
+  },
+  multipath: {
+    pre: [{
+      name: 'activate multipath',
+      body: |||
+        #!/bin/bash
+        if ! systemctl status multpathd ; then
+          echo 'Activating multipath'
+          systemctl start multipathd.socket
+          systemctl start multipathd
+        fi
+      |||,
+    }]
+  },
+  wipefs: {
+    pre: [
```

Comment:
we should be able to reference the array item, so we can use more than one `pre`.